gensim/models/word2vec.py

```diff
@@ -387,6 +409,12 @@ def __init__(
 
         `iter` = number of iterations (epochs) over the corpus.
 
+        `trim_rule` = vocabulary trimming rule, specifies whether certain words should remain
+         in the vocabulary, be trimmed away, or handled using the default (discard if word count < min_count).
+         Must be a callable that accepts a tuple (word, count, min_count) and returns
+         either word2vec.RULE_DISCARD, word2vec.RULE_KEEP or word2vec.RULE_DEFAULT.
+         Note: the rule is NOT serialized.
```

Comment:
"NOT serialized" is too obscure. How about "the rule, if given, is only used prune vocabulary during `build_vocab()` and is not stored as part of the model."?